.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,7 +2,7 @@
 name: Feature request
 about: Request new or improved functionality
 title: ''
-labels: ''
+labels: engineering, Request
```

Comment:
let's make this label `feature request` rather than `Request`

Reply:
Hi @Pomax, thanks for the feedback, I have made the requested changes!